lib/ansible/utils/collection_loader/_collection_finder.py

```diff
@@ -224,6 +230,52 @@ def find_module(self, fullname, path=None):
             # TODO: log attempt to load context
             return None
 
+    def find_spec(self, fullname, path, target=None):
```

Comment:
I think I'd like to see this consolidated with `find_module`. They are about 99% the same.  My recommendation is to drop everything from `find_module` and then do:

```python
    def find_module(self, fullname, path=None):
        spec = self.find_spec(fullname, path=path)
        if spec:
            return spec.loader
        return None
```

Reply:
Yep

Reply:
Done